src/index.html

```diff
@@ -2,15 +2,53 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="get"
+    data-qa="big">
```

Comment:
- each level of nesting requires 2 indents
- opening brace and the closing brase should be at the same level of indentation if you move attributes to the new line
```suggestion
      action="#"
      method="get"
      data-qa="big">
```